### PR TITLE
Fix mermaid diagrams in PDFs.

### PR DESCRIPTION
**Pull request by wmat**
Added mermaid-cli to package.json.
Set mermaid code wrapper to [mermaid, ,svg].

### CODE REVIEW

File: specification/07-theory_operations.adoc [hunk 1]
```diff
@@ -135,7 +135,7 @@ initiated CoVE-IO `COVH` calls that require SPDM requests to be sent to the devi
 DOE mailbox follows the flow described below:
 
 .SPDM Flow With CoVE
-[source,mermaid]
+[mermaid, ,svg]
```
Comment (sameo) [inline specification/07-theory_operations.adoc:138]: Did you mean `mermaid, svg`?

**Comment by sameo** [comment]
Thanks for the PR @wmat !
Unfortunately, the generated PDF diagrams are wrong, i.e. we only see the mermaid code. Is that because of https://github.com/riscv-non-isa/riscv-ap-tee-io/pull/62#discussion_r1461095838 ?

File: specification/07-theory_operations.adoc (hunk 1 above)
Comment (wmat) [reply specification/07-theory_operations.adoc:138]: Nope, I was following the asciidoctor diagram block syntax: https://docs.asciidoctor.org/diagram-extension/latest/blocks/

Comment (sameo) [reply specification/07-theory_operations.adoc:138]: Unfortunately, that seems to break the markdown rendering, and does not fix the PDF generation neither.

**Comment by sameo** [comment]
#118 should fix that one.

**Pull request closed without merging**